README.md

```diff
@@ -175,14 +175,28 @@ kubectl delete --ignore-not-found customresourcedefinitions \
 #### Running *end-to-end* tests on local kind cluster:
 
 1. `kind create cluster --image=kindest/node:<latest>`. e.g `v1.23.0` version.
+
+> Note: In case you are running kind on MacOS using podman, create podman machine with at least 4 CPUs and 8GiB memory:
```

Comment:
Is the 4CPU and 8GiB memory a requirement, or just an example?

Reply:
Using 4 CPUs on my environment end to end tests started passing. If it is a requirement or just an accidental value I don't know. But having just one or two CPUs podman machine makes end to end tests timeout because operator fails to deploy complaining on lack of CPU resources. The value for 8GiB memory I got from some googled documentation  (recommended value, minimal was 4GB), I don't have at hand now, but can google it again in case it's required.

Reply:
Ok, thanks for the detailed explanation! Maybe we can just reword it to say that 4 CPUs and 8GiB is recommended for podman machine in general, so that people don't think this is what the operator needs :)

Reply:
Fair enough, will reword it then.